src/app.rs

```diff
@@ -49,4 +49,12 @@ pub fn build_app() -> App<'static, 'static> {
                 .long("tree")
                 .help("Recurse into directories and present the result as a tree"),
         )
+        .arg(
+            Arg::with_name("date")
+                .long("date")
+                .possible_value("normal")
```

Comment:
I would vote for replacing "normal" by "date". It make more logic to me to have date/relative instead of default/relative. Another proposition would be "classic".

What do you think of it?

Reply:
`date` sounds good. `classic` can be a bit ambiguous.